prov/rxm/src/rxm_conn.c

```diff
@@ -239,11 +243,18 @@ static void *rxm_conn_event_handler(void *arg)
 		case FI_CONNECTED:
 			FI_DBG(&rxm_prov, FI_LOG_FABRIC,
 			       "Connection successful\n");
+			fastlock_acquire(&rxm_ep->util_ep.cmap->lock);
 			cm_data = (void *)entry->data;
+			if (rd - sizeof(*entry))
+				((struct util_cmap_handle *)
+					entry->fid->context)->remote_key =
+						cm_data->conn_id;
```

Comment:
Why do we need to set this? Can't we just call rxm_conn_handle_postponed_tx_op after ofi_cmap_process_connect which already sets remote_key?

Reply:
Sure.

Sorry I forgot to remove this confusing thing.